data/blog/how-we-improved-performance-score-accuracy.md

```diff
@@ -0,0 +1,52 @@
+images: ['/images/how-we-improved-performance-score-accuracy/hero.jpg']
+layout: PostLayout
+canonicalUrl: performance-score-improvements
+authors: ['edwardgou']
+---
+
+Last year in November, we released the Web Vitals module for Sentry's Performance product. Aside from helping users monitor and improve their Web Vitals, this new module also introduced [Sentry Performance Scores](https://docs.sentry.io/product/performance/web-vitals/#performance-score) for all web browser applications. As a quick reminder, Performance Scores are used to condense multiple Web Vitals and their respective thresholds into an overall score from 0-100 that rates an app's perceived performance, based on real user data for Web Vitals.
+
+Today, we are making updates to how Performance Scores are calculated. These updates will improve the accuracy of these scores by bringing them closer to what users experience on your web apps. To understand how these changes will impact your Performance Scores, it's useful to understand how they used to be calculated. Here's a brief overview:
+
+1. Your users visit your webpage which generate pageload transactions. These transactions contain [Web Vitals](https://docs.sentry.io/product/performance/web-vitals/) (LCP, FCP, FID, TTFB, CLS) and get sent to Sentry.
+2. In the Sentry Web Vitals module, each web vital is aggregated to get the p75 value.
+3. The p75 value of each Web Vital is then put through a non-linear function which generates individual Web Vital component scores. Each component score is then statically weighted and summed to produce an overall Performance Score out of 100 for your web app. You can find out more about how these are calculate [here](https://docs.sentry.io/product/performance/web-vitals/#performance-score).
+
+While the above approach gives some of the signal that we're looking for, there are some drawbacks that affect how accurately our Performance Scores reflect what users experience:
+
+- Performance Scores can be skewed based on outlier pageloads if the Web Vitals are extreme enough.
+- Some Web Vitals may be reported less frequently, depending on [browser support](https://docs.sentry.io/product/performance/web-vitals/#browser-support). These Web Vitals end up overrepresented in Performance Scores because we use static weights.
+
+Ideally, our Performance Scores should feel true to what users experience on our web app. If our web app is fast and responsive, our Performance Scores should positively reflect that. Additionally, Performance Scores should show a consistent relationship when drilling down from web app, to pages, to pageloads.
+
+To demonstrate the issues mentioned above, let’s consider a scenario where we have three pageload transactions, with the following **LCP** values and thresholds:
+
+```
+Good    | Meh     | Poor
+-----------------------------
+<1200ms | <2400ms | >= 2400ms
+
+300ms  -> Good
+400ms  -> Good
+8000ms -> Poor
+```
```

Comment:
This part confused me a bit, maybe worth breaking up! 

1. It's not necessarily obvious that a measure is mapped to a rating like "good", the post doesn't describe that
2. The table and the examples flow together. If you keep the table markup separate, it'll render as an HTML table, which helps. Then, you can write something like "Let's say three page load transactions come in with the following LCP values, and corresponding ratings

Reply:
I opted to remove the `Good` `Meh` and `Poor` thresholds here, since they're mostly part of the UI and not actually involved in the score calculation at all.